packages/manager/modules/pci/src/projects/project/storages/containers/containers.constants.js

```diff
@@ -96,6 +96,21 @@ export const OPENIO_DEFAULT_REGION = 'SBG';
 
 export const OPENIO_PRESIGN_EXPIRE = 3600;
 
+export const CONTAINER_GUIDES = [
+  {
+    id: 'rclone',
+    link: 'https://docs.ovh.com/fr/storage/sync-rclone-object-storage/',
+  },
+  {
+    id: 'veeam',
+    link: 'https://docs.ovh.com/fr/storage/s3/veeam/',
+  },
+  {
+    id: 's3-limitation',
+    link: 'https://docs.ovh.com/fr/storage/s3/limitations/',
+  },
+];
+
```

Comment:
Is there only need for a FR guide?

Reply:
No, you have right 👍 .
We are waiting for others links.

Reply:
Done